RecoPixelVertexing/PixelLowPtUtilities/bin/BuildFile.xml

```diff
@@ -0,0 +1,3 @@
+<bin file="ClusterShapeAnalyzer.cc" name="clusterShapeAnalyzer">
+	<use name="root"/>
```

Comment:
typically we use the pieces of root that are needed rather then the entire package (which has lots of un-desirable stuff for most binaries)